doc/examples/contextual_enhancement.py

```diff
@@ -188,17 +190,17 @@
 csd_sf_enh_sharp = sh_to_sf(csd_shm_enh_sharp, sphere, sh_order=8)
 
 # Normalize the sharpened ODFs
-csd_sf_enh_sharp = csd_sf_enh_sharp * np.amax(csd_sf_orig)/np.amax(csd_sf_enh_sharp)
+csd_sf_enh_sharp = csd_sf_enh_sharp * np.amax(csd_sf_orig)/np.amax(csd_sf_enh_sharp) * 1.25
```

Comment:
Why the different sharpening now? I'm too lazy to regenerate the pictures at the moment.

Oh it seems like it was in the sphere viz function before I guess while looking below.